### ISSUE TITLE
NSBundle.h error: function does not return NSString

### ISSUE THREAD

**Issue by krottv** [icerockdev/moko-resources#247]
After updating to XCode 13 I get errors when executing ./gradlew -p resources build publishToMavenLocal

```
Exception in thread "main" java.lang.Error: /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS15.0.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSBundle.h:91:143: error: function does not return NSString
        at org.jetbrains.kotlin.native.interop.indexer.UtilsKt.ensureNoCompileErrors(Utils.kt:192)
        at org.jetbrains.kotlin.native.interop.indexer.IndexerKt.indexDeclarations(Indexer.kt:1206)
        at org.jetbrains.kotlin.native.interop.indexer.IndexerKt.buildNativeIndexImpl(Indexer.kt:1195)
        at org.jetbrains.kotlin.native.interop.indexer.IndexerKt.buildNativeIndexImpl(Indexer.kt:1191)
        at org.jetbrains.kotlin.native.interop.gen.jvm.DefaultPlugin.buildNativeIndex(Plugins.kt:31)
        at org.jetbrains.kotlin.native.interop.gen.jvm.MainKt.processCLib(main.kt:272)
        at org.jetbrains.kotlin.native.interop.gen.jvm.MainKt.interop(main.kt:76)
        at org.jetbrains.kotlin.cli.utilities.InteropCompilerKt.invokeInterop(InteropCompiler.kt:45)
        at org.jetbrains.kotlin.cli.utilities.MainKt.mainImpl(main.kt:38)
        at org.jetbrains.kotlin.cli.utilities.MainKt.main(main.kt:60)
```

**Comment by Alex009** [comment]
Something in k/n not support new xcode. Try workaround from this issue https://youtrack.jetbrains.com/issue/KT-48807

**Comment by Alex009** [comment]
fixed in #262 and will be available in 0.17.3
